backpack/core/derivatives/convnd.py

```diff
@@ -194,15 +183,27 @@ def __same_conv_weight_jac_t(self, module, mat, sum_batch):
             size = module.weight.shape[2 + dim]
             grad_weight = grad_weight.narrow(axis, 0, size)
 
-        sum_dim = "" if sum_batch else "n,"
-        # separate group axes from vectorization axes
-        eingroup_eq = "vngio,{}->v,{}go,i,{}".format(dims, sum_dim, dims)
-
-        return eingroup(
-            eingroup_eq,
-            grad_weight,
-            dim={"g": G, "v": V, "n": N, "i": C_in // G, "o": C_out // G},
-        )
+        if sum_batch:
+            return reduce(
+                grad_weight,
+                "(v n g i o) ... -> v (g o) i ...",
+                "sum",
+                g=G,
+                v=V,
+                n=N,
+                i=C_in // G,
+                o=C_out // G,
```

Comment:
Suggestion: Define 
```
dim={"g": G, "v": V, "n": N, "i": C_in // G, "o": C_out // G}
```
then replace these lines by `**dim`. `dim` can be recycled in the `rearrange` call below, too.